pages/planetusermanual.md

```diff
@@ -218,6 +218,123 @@ Collections can be used by Nation Managers to categorize materials by topic for
 
 7. You will be directed to the same screen as the creation page. Change any of the information which should be updated and click “Save.” You can also delete or nest (sub-categorize) the Collection.
 
+## Courses Overview
+### What is a Course?
+A Course allows Leaders to walk Learners through specific Resources and exercises to learn new skills.  Leaders can create Courses by grouping and ordering Resources and allowing Learners to upload papers or take Quizzes.  
+
+### How do I create a Course?
+To create a Course, log in and start from the Home page.
+* Go to the *“Courses”* tab in the top menu and then select *“Add Course.”*
```

Comment:
There is an asterisk at the end of line 227 on [rawgit](https://rawgit.com/sarah-lu102/sarah-lu102.github.io/manualcourses/#!pages/planetusermanual.md#Courses_Overview) but looks good on [github markdown file](https://github.com/sarah-lu102/sarah-lu102.github.io/blob/manualcourses/pages/planetusermanual.md)